ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java

```diff
@@ -313,10 +316,11 @@ public static Bytes32 get_active_index_root(BeaconState state, UnsignedLong epoc
         get_current_epoch(state)
                 .plus(UnsignedLong.valueOf(ENTRY_EXIT_DELAY))
                 .compareTo(epoch.plus(UnsignedLong.valueOf(LATEST_INDEX_ROOTS_LENGTH)))
-            < 0);
+            < 0,
+        "checkArgument threw and exception in get_active_indesx_root()");
     checkArgument(
-        epoch.compareTo(get_current_epoch(state).plus(UnsignedLong.valueOf(ENTRY_EXIT_DELAY)))
-            <= 0);
+        epoch.compareTo(get_current_epoch(state).plus(UnsignedLong.valueOf(ENTRY_EXIT_DELAY))) <= 0,
+        "checkArgument threw and exception in get_active_index_root()");
```

Comment:
Did you forget to change the string here?

Reply:
Lol doh

Reply:
actually, if you expand the code.  you can see that it was part of  get_active_index_root()